tests/test_fork_provider.py

```diff
@@ -30,19 +33,6 @@ def create_mainnet_fork_provider(network_api):
         provider_settings={},
     )
 
-@alchemy_xfail
-def test_request_timeout(mainnet_fork_provider, config, network_api):
```

Comment:
I moved this test by the other tests, so it is not in the middle of the fixtures